pkg/generic/thrift/thrift.go

```diff
@@ -29,7 +31,7 @@ const (
 
 // MessageReader read from thrift.TProtocol with method
 type MessageReader interface {
-	Read(ctx context.Context, method string, in thrift.TProtocol) (interface{}, error)
+	Read(ctx context.Context, method string, msgType remote.MessageType, dataLen int, in thrift.TProtocol) (interface{}, error)
```

Comment:
Break change

Reply:
why do we need `msgType remote.MessageType, dataLen int` here?

Reply:
@AsterDY In the old logic, generic Unmarshal() passes `remote.Message` to generic Reader when initializing the Reader, and Read() gets the msgType and dataLen from the remote.Message.
However, now we skip the generic Unmarshal(), so there's no way to pass the message unless we change the Read interface